### PR TITLE
Feature/remote syntax

### PR DESCRIPTION
**Pull request by meganrm**
Problem
=======

Using `autoPACKserver/` as a keyword is confusing and breakable

Solution
========
changed the syntax to `github:` and `firebase:` for our two remote dbs. 
with @rugeli 

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)
* Breaking change (fix or feature that would cause existing functionality to not work as expected)
* This change requires a documentation update

Steps to Verify:
----------------
1.  `conda activate autopack`
2.  `pack -r  github:recipes/NM_Analysis_FigureB1.0.json  -c packing-configs/run.json`
Screenshots (optional):
-----------------------
Show-n-tell images/animations here

### CODE REVIEW

File: cellpack/autopack/Compartment.py [hunk 1]
```diff
@@ -1331,16 +1331,6 @@ def BuildGrid_scanline(
                         if inside:
                             insidePoints.append(ptInd)
                             idarray.itemset(ptInd, -number)
-                    p = (ptInd / float(len(grdPos))) * 100.0
-                    if (ptInd % 1000) == 0 and autopack.verbose:
-                        helper.progressBar(
```
Comment (meganrm) [inline cellpack/autopack/Compartment.py:1336]: removed this `verbose` setting. If we want to we can replace with the logger

File: cellpack/autopack/__init__.py [hunk 2]
```diff
@@ -132,24 +126,10 @@ def url_exists(url):
 
 usePP = False
 helper = None
-LISTPLACEMETHOD = ["jitter", "spheresBHT"]
-try:
-    from panda3d.core import Mat4  # noqa: F401
-
-    LISTPLACEMETHOD = ["jitter", "spheresBHT", "pandaBullet"]
-except ImportError:
-    LISTPLACEMETHOD = ["jitter", "spheresBHT"]
```
Comment (meganrm) [inline cellpack/autopack/__init__.py:141]: we weren't using these anywhere

File: cellpack/autopack/__init__.py [hunk 3]
```diff
@@ -132,24 +126,10 @@ def url_exists(url):
...
-
 ncpus = 2
 checkAtstartup = True
 testPeriodicity = False
 biasedPeriodicity = None  # [1,1,1]
-fixpath = False
-verbose = 0
-messag = """Welcome to autoPACK.
-Please update to the latest version under the Help menu.
-"""
```
Comment (meganrm) [inline cellpack/autopack/__init__.py:152]: reducing globals

File: setup.py [hunk 4]
```diff
@@ -25,6 +25,7 @@
 analysis_requirements = [
     "tqdm>=4.64.1",
     "scikit-learn>=1.1.3",
+    "seaborn>=0.12.1",
```
Comment (mogres) [inline setup.py:28]: Thanks for updating this requirement

**Review by mogres — APPROVED** [review]
Lots of very good changes. Looks good to me!

**Pull request merged**